apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java

```diff
@@ -122,12 +123,14 @@ int getActivityType() {
     public void openAudio() throws IOException {
         super.openAudio();
         mShouldSetStreamControlByAttributes.setEnabled(false);
+        mShouldDisableForCompressedFormat = StreamConfiguration.isCompressedFormat(
+                mAudioOutTester.getCurrentAudioStream().getFormat());
     }
 
-    private void configureChannelBoxes(int channelCount) {
+    private void configureChannelBoxes(int channelCount, boolean shouldDisable) {
         for (int i = 0; i < mChannelBoxes.length; i++) {
             mChannelBoxes[i].setChecked(i < channelCount);
-            mChannelBoxes[i].setEnabled(i < channelCount);
+            mChannelBoxes[i].setEnabled(!shouldDisable && (i < channelCount));
```

Comment:
Should we change setEnabled to setVisible?

Reply:
Should be fine with visible but not enabled so that people know how many channels are used.